.github/ISSUE_TEMPLATE/3_plugin_request.yml

```diff
@@ -33,32 +33,51 @@ body:
           required: true
         - label: "[I have checked the list of open and recently closed plugin requests](https://github.com/streamlink/streamlink/issues?q=is%3Aissue+label%3A%22plugin+request%22)"
           required: true
+  - type: checkboxes
+    attributes:
+      label: Collaboration
+      options:
+        - label: My request is genuine, in the best interest of the project, and I have access to the stream(s) in my web browser
```

Comment:
Does this need to be expanded to "without the usage of a VPN"?

Reply:
I was thinking about it, but there's the access restriction question down below

Reply:
Works for me! Can always update it if needed.